models/biocro/R/met2model.BIOCRO.R

```diff
@@ -30,7 +30,7 @@ met2model.BIOCRO <- function(in.path, in.prefix, outfolder, overwrite=FALSE, ...
     tmp.met <- load.cfmet(met.nc, lat = lat, lon = lon, start.date = start.date, end.date = end.date)
     metlist[[file]]     <- cf2biocro(tmp.met)
   }
-  met <- rbindlist(metlist)
+  met <- rbindlist(metli)
```

Comment:
@robkooper Good catch. I forgot that I disabled the package tests 56d80b19c843b438e75344913693bf95078fc60a and I also neglected to write an issue. But I'll consider this #239 and fix.